demos/random_forest/new_coordinator.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2021 Fedlearn authors.
```

Comment:
Add a todo of refactoring the whole file once update is done, otherwise it will be confusing to have multiple files with similar names.